src/nuget/Microsoft.Build.Tasks.Core.nuspec

```diff
@@ -19,8 +19,9 @@
       <group targetFramework="dotnet">
         <dependency id="Microsoft.Build.Framework" />
         <dependency id="System.IO.FileSystem.DriveInfo" version="4.0.0-beta-23302" />
-        <dependency id="Microsoft.Win32.Primitives" version="4.0.1-beta-23516" />
         <dependency id="Microsoft.Win32.Registry" version="4.0.0-beta-23516" />
```

Comment:
Yes, we should, thanks for the catch. These are auto-generated, so the fix should be to remove the references from `project.json`. #439